src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Layouts/Stack/BitStackDemo.razor.scss

```
@@ -1,7 +1,74 @@
     .item {
         color: white;
         padding: 0.5rem;
+        white-space: nowrap;
         background-color: #0078d4;
     }
+
+    // The outer stack of the examples that show a stack behaving as a child of another one. It is only
+    // an outline, so what is being demonstrated is the inner stacks and not the container.
+    .host {
+        padding: 0.5rem;
+        // What no longer fits is what several of these examples are about, so the box that is being run
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)